pjlib/src/pj/log.c

```diff
@@ -284,8 +284,8 @@ static void suspend_logging(int *saved_level)
 #if PJ_HAS_THREADS
     if (thread_suspended_tls_id != -1) 
     {
-	pj_thread_local_set(thread_suspended_tls_id, 
-			    (void*)(pj_ssize_t)PJ_TRUE);
+	static int suspend_log = PJ_TRUE;
```

Comment:
perhaps make the type `pj_ssize_t` so it's the same as the previous code?

Reply:
Don't think it should matter since it's a pointer being passed.
I guess it's related to this: https://sourceware.org/bugzilla/show_bug.cgi?id=27714
I checked how this was used and when fetching thread_suspended_tls_id, it only checks if the pointer is there, it doesn't really check the value.

Reply:
Reading the reference, it doesn't seem that everybody is happy about the latest change, so perhaps they'll go back to the old way?

There's another one below though, in `resume_logging()`.